usaspending_api/etl/elasticsearch_loader_helpers/__init__.py

```diff
@@ -0,0 +1,54 @@
+from usaspending_api.etl.elasticsearch_loader_helpers.fetching_data import (
+    configure_sql_strings,
+    download_db_records,
+    get_updated_record_count,
+)
+from usaspending_api.etl.elasticsearch_loader_helpers.delete_data import (
```

Comment:
Just for consistency, you have `fetching_data` and `indexing_data` ... to follow suit this one should be `deleting_data`. Although I'm more partial to flipping the other two to singular (`index_data`, `fetch_data` along with `delete_data`)

Reply:
Also, _could_ follow ETL semantics and be `extract_data`, `load_data`, `delete_data` ... 

And then `ElasticsearchRunner` might fit better if renamed to `ElasticsearchEtl`

Reply:
Open to any and all naming suggestions.

Reply:
@kwhickey Pushed a commit with new names. Didn't change `fetch` -> `extract` but I can if we still feel that is preferable.